backend/src/controllers/giftsController/index.ts

```diff
@@ -54,4 +55,17 @@ export default class GiftsController {
     res.json(newGift)
   }
 
+  @Delete('/:id', [AuthMiddleware])
+  public async deleteGift(
+    @QueryParam('id') id: string,
+    @ResponseDecorator() res: Response
+  ) {
+    const deletedGift = await this.giftService.deleteGift(id)
+
+    if (!deletedGift) {
```

Comment:
I think we should add check if gift is in user's team
`gitft.team_id === user.team_id`